components/Post/PostContent.tsx

```diff
@@ -31,9 +37,14 @@ const PostContent = ({ post }: Props) => {
       </Head>
 
       <div className="blogpost-image">
+        {isPostADraft(post) ? <DraftPostMark /> : null}
+        {isPostInTheFuture(post) === false ? <FuturePostMark /> : null}
```

Comment:
@ntorbinskiy 🤯

If post NOT in the future, then show FuturePostMark

It makes no sense

Reply:
And is there a specific reason you use ternary instead of && operator?